### ISSUE TITLE
[BUG] percentage is wrong in case of multiple puchases of same asset in different time and with different costs??!?!?

### ISSUE THREAD

**Issue by posta246** [ghostfolio/ghostfolio#4041]
(sorry me for my bad english :) )

**Bug Description**

In the case of purchases of the same asset over time (e.g. ETFs) but at an obviously different unit cost, the overall profit percentage seems incorrect to me.

Just add different purchases in different time for the same item, using different unit price. Look at the testing picture I build: how is possible that I earned 44%?

**Expected behavior**

I would have expected the final average percentage of earnings to be equal to the weighted average of the earnings of each purchase

**Screenshots**

![Schermata a 2024-11-11 12-09-31](https://github.com/user-attachments/assets/0305678e-e55d-4f76-bbf9-82aaacbaa1e7)

**Environment**

- Ghostfolio Version: latest
- Self-hosted
- Experimental Features enabled or disabled: i think disable....
- Browser: firefox
- OS: linux, android

**Comment by Se7enair** [comment]
I can confirm that. Two ETFs, different buy activities, performance is too high. 
Performance should be around 10%
![grafik](https://github.com/user-attachments/assets/a9f410ac-e1f1-420e-b02e-f64a9c80f90b)

**Comment by karter16** [comment]
ghostfolio is littered with these sorts of inaccuracies unfortunately. This thread has a summary of the most concerning ones: https://github.com/ghostfolio/ghostfolio/discussions/3666

**Comment by posta246** [comment]
I'm evaluating other tools or writing a stupid one via ollama, cause I just would like to have a daily recap.

**Comment by FlamebergUA** [comment]
Hey all, just checking if this has been resolved? It seems to be a great piece of software, but this is a serious bug - I'm sure most of users invest in ETFs constantly, so there's an expected use case of having multiple transactions. Incorrect reporting means this can't be used for calculating taxes, for example.

**Comment by posta246** [comment]
I wrote a stupid bot the send telegram message contains the recap I need. Ghostfolio disinstalled.

**Comment by karter16** [comment]
yup - devs appear to have no interest in fixing the fundamental calculation flaws in Ghostfolio - glad all the packages are up to date though ;-)

**Comment by FlamebergUA** [comment]
> yup - devs appear to have no interest in fixing the fundamental calculation flaws in Ghostfolio - glad all the packages are up to date though ;-)

sigh... Back to Excel it is then, lol.

**Comment by Sjohn21** [comment]
See discussion: #2960 
It seems to be calculated as Time-weighted performance.
There seems to have been a simpler calculation of performance up till release [2.32.0](https://github.com/ghostfolio/ghostfolio/releases/tag/2.32.0)

I myself would also rather have to see this simpler calculation [eg. (Current value - Investment value) / Investment value ]
However, I lack the skills to implement a performance choice switcher.
